sql/handler.h

```diff
@@ -403,6 +403,7 @@ enum enum_alter_inplace_result {
 #define HA_CREATE_TMP_ALTER     8U
 #define HA_LEX_CREATE_SEQUENCE  16U
 #define HA_VERSIONED_TABLE      32U
+#define HA_VTMD                 64U
```

Comment:
Looks like this flag is used only on VTMD table creation. Not even on .frm load. Is it really needed?